### PR TITLE
Pagination search   am

### PR DESCRIPTION
**Pull request by Arcziwald**
I got the pagination activity working properly as well as loading pages in /people
I ran the correct pagination activity in /search
The search activated after the entered phrase has been set
Fixes related to loading the /people home page and loading between pages using pagination
I styled the main navigation buttons when searching

Of course, I will remove all console.log before the merge :-)

### CODE REVIEW

File: src/common/Navigation/index.js [hunk 1]
```diff
@@ -8,13 +9,19 @@ import {
...
         <StyledDiv>
           <StyledLogo />
-          <StyledNavLink to="/movies">Movies</StyledNavLink>
-          <StyledNavLink to="/people">People</StyledNavLink>
+          <StyledNavLink to="/movies" className={location.pathname === "/movies" ? "active" : ""}>
+            Movies
+          </StyledNavLink>
+          <StyledNavLink to="/people" className={location.pathname === "/search" ? "active" : ""}>
+            People
+          </StyledNavLink>
```
Comment (WoytekMig) [inline src/common/Navigation/index.js:24]: great, that you added this ! :)

File: src/features/MoviesBrowser/SearchResult/index.js [hunk 2]
```diff
@@ -38,24 +48,34 @@ const SearchResult = ({ onPageChange, currentPage }) => {
...
         }
       />
       {isLoading ? (
         <Loading />
-      ) : isError || !data || data.results.length === 0 ? (
+      ) : isError ||
+        !dataLoaded ||
+        !data ||
+        !data.results ||
+        data.results.length === 0 ? (
```
Comment (KrystianZybura) [inline src/features/MoviesBrowser/SearchResult/index.js:62]: Isn't it a bit too complicated? 😅

**Review by KrystianZybura — APPROVED** [review]
Works well, great job 😎

**Review by WoytekMig — APPROVED** [review]
high five for both of us ! :D

File: src/features/MoviesBrowser/SearchResult/index.js (hunk 2 above)
Comment (WoytekMig) [reply src/features/MoviesBrowser/SearchResult/index.js:62]: I made an issue - agree, that it should have a little less restrictions :D

**Pull request merged**